crates/orchestrator/src/network/node.rs

```diff
@@ -159,7 +159,11 @@ impl NetworkNode {
 
     /// Get the rpc client for the node
     pub async fn rpc(&self) -> Result<RpcClient, subxt::Error> {
-        RpcClient::from_url(&self.ws_uri).await
+        if subxt::utils::url_is_secure(&self.ws_uri)? {
+            RpcClient::from_url(&self.ws_uri).await
+        } else {
+            RpcClient::from_insecure_url(&self.ws_uri).await
+        }
```

Comment:
Nice catch, we use the same logic [here](https://github.com/paritytech/zombienet-sdk/blob/fd43dba31990aeb47fa30bcb1d4b20fb7e1afbbe/crates/orchestrator/src/network/node.rs#L188-L192), maybe we can abstract in some way and allow also to get a client/rpc_client by just passing the url.

Reply:
Added `ClientFromUrl` trait and `get_client_from_url()` helper method, which allows to get `RpcClient` or `OnlineClient`